src/number-formatting/number-formatting.js

```diff
@@ -72,14 +72,7 @@ NumberFormatting.prototype.format = function(num, decimals) {
  * @param {number} [maxDecimals] - The maximum number of decimals to display after the decimal point.
  * @returns {string}
  */
-NumberFormatting.prototype.formatNoRounding = function(num, minDecimals, maxDecimals) {
-    if (!minDecimals) {
-        minDecimals = 0;
-    }
-    if (!maxDecimals) {
-        maxDecimals = 8;
-    }
-
+NumberFormatting.prototype.formatNoRounding = function(num, minDecimals = 0, maxDecimals = 8) {

```

Comment:
Please make sure We don't have existing behaviour which might rely on old logic ie. passing of null/NaN as maxDecimals. 

If we are not sure, it would be safer to have uglier option of explicit check against null, undefined and NaN.